@here/harp-mapview/lib/Utils.ts

```diff
@@ -174,12 +169,9 @@ export namespace MapViewUtils {
         mapView: MapView,
         deltaAzimuthDeg: number,
         deltaTiltDeg: number,
-        maxTiltAngleRad = Math.PI / 2
+        maxTiltAngleRad = maxTiltAngleAllowed
```

Comment:
Shouldn't this be < 90 degrees. If we allow 90 degrees we will get div. by 0 errors

Reply:
It is smaller then 90 degrees, actually set in MapView (const MAX_TILT_ANGLE) as 89 deg.